sqlglot/tokens.py

```diff
@@ -525,6 +525,8 @@ def _quotes_to_format(
             if " " in key or any(single in key for single in klass.SINGLE_TOKENS)
         )
 
+        klass._use_rs_tokenizer = USE_RS_TOKENIZER
+
```

Comment:
We shouldn't need to change the metaclass logic for this PR. Let's discuss in Slack.

Reply:
Actually this line can be removed since `USE_RS_TOKENIZER` is a top level constant